docs/index.rst

```diff
@@ -97,6 +97,18 @@ like buffering:
     >>> secondpart = b.read()
 
 
+The response can be treated as a file-like object.
+A file can be downloaded directly to a local file in a context without
+being saved in memory.
+
+.. doctest ::
```

Comment:
`.. doctest ::` is a comment in reStructuredText, not a directive. It should be a directive.